polaris-core/src/main/java/org/apache/polaris/core/PolarisConfiguration.java

```diff
@@ -103,6 +103,22 @@ public static <T> Builder<T> builder() {
               .defaultValue(false)
               .build();
 
+  // Config key for whether to skip credential-subscoping indirection entirely whenever trying
```

Comment:
Can we move this into the description? The description should ideally be detailed enough to understand the config without an additional comment explaining it.